charts/consul/templates/server-acl-init-job.yaml

```diff
@@ -251,7 +251,7 @@ spec:
                 {{- end }}
 
                 {{- if .Values.controller.enabled }}
-                -create-controller-token=true \
+                -enable-controller=true \
```

Comment:
how about `-controller=true`? It seems similar to patterns we have for partitions.